multi-agent/adapters/outbound/mongo/builtin_user_config_repository.py

```
@@ -0,0 +1,108 @@
+            )
+        except pymongo.errors.DuplicateKeyError:
+            doc = self.col.find_one_and_update(
+                filter_doc, update_doc,
+                upsert=True,
+                return_document=pymongo.ReturnDocument.AFTER,
+            )
+        except pymongo.errors.PyMongoError as exc:
+            logger.error(
+                "Mongo error saving BuiltinUserConfig for resource=%s, identity=%s: %s",
+                config.resource_id, config.identity_key, exc,
+            )
```

Ruff (0.16.1)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)